transport-msg.tex

```diff
@@ -868,16 +875,32 @@ \subsubsection{Overview}
 & 12 & ... & Configuration data \\
 \hline
 Answer & 0 & 4 & Configuration offset in bytes\\
-& 4 & 4 & Number of bytes \\
-& 8 & 4 & Configuration generation count \\
+& 4 & 4 & Number of bytes, or 0 if rejected \\
+& 8 & 4 & New Configuration generation count \\
 & 12 & ... & Configuration data \\
 \hline
 \end{tabular}
 
+If the request's generation count matches the device's generation count when
+this message is processed by the device, then the specified data will be updated
+and the resulting data will be returned. The resulting data may not match the
+request data as the device may implement read-only parts of the config area or
+have other constraints on the settings. A \msgref{SET_CONFIG} with 0 for
+"Number of bytes"  in the request is illegal.
+
+If the request and current configuration generation counts do not match, no
+change to the configuration data is made and the response is given with the
+current generation counter and a 0 for "Number of bytes".  As signaled by
+"Number of bytes", no configuration data is returned in the response. The
+driver will likely know the new configuration state and generation count by
+processing \msgref{EVENT_CONFIG} messages.  If this is not the case it should
+use \msgref{GET_CONFIG} to come up to date.  The driver may then retry the
+\msgref{SET_CONFIG} operation if still desired.
+
```

Comment:
We are repeating more or less exactly what was explained in the configuration count chapter before.
maybe we should just make a reference to this chapter instead of repeating

Reply:
I don't like the duplication but I greatly prefer the details of a message to be with the message.  I would vote we skinny the other section down and keep this one.

Reply:
I did replace this section with a reference.  It is enough here to describe the rejection field value and the message table does that.